### DISCUSSION TITLE
Thunderbird error

### DISCUSSION THREAD

**Discussion by lexesv** [emersion/maddy#428] [Q&A]
![2021-12-26_18-51](https://user-images.githubusercontent.com/15149258/147415137-86b3af7b-2c88-452b-b736-ec45ff1dc927.png)

The user is created, a Emails go to the "messages" directory.

Config: https://pastebin.com/4jgzkwRn

**Comment by foxcpp** [comment]
What is in server logs?

**Comment by lexesv** [comment]
There are no errors in the logs. Debug mode is on.

**Reply by lexesv** [reply]
The issue was solved by adding "tls &local_tls" to the imap and submission sections.
